src/functions/indBallRank.jl

```diff
@@ -41,7 +41,11 @@ function prox!{T <: Real}(f::IndBallRank, x::AbstractArray{T,2}, y::AbstractArra
     for j = 1:size(x,2)
       y[i,j] = 0.0
       for k = 1:f.r
-        y[i,j] += svdobj[:U][i,k]*svdobj[:S][k]*svdobj[:Vt][j,k]
+          if VERSION > v"0.5"
```

Comment:
Also: I noticed that many (if not all) PRs for METADATA.jl come with a nice comment by the author of the PR pointing to diffs, Travis builds and so on. I guess that is generated automatically when one publishes a package release directly from the Julia REPL, right?